lib/galaxy/exceptions/utils.py

```diff
@@ -0,0 +1,32 @@
+from galaxy.exceptions import error_codes
+
+
+def api_error_to_dict(trans, **kwds):
```

Comment:
I do on the other hand, really like this. This is a nice enhancement IMO.

Reply:
Marius' idea, it has to be ;)